sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -41,13 +41,17 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
+  private def castWithDefaultTZ(v: Any, targetType: DataType): Cast = {
```

Comment:
shall we just change the default value of `timeZoneId` parameter in `def cast` to GMT?